docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,141 @@
+# Developers Clear Containers 3.0 Install
+
+This guide is not intended for end-users. Instead, this guide provides
+instructions for any developers eager to try Clear Containers 3.0 and who
+want to build Clear Containers from the source code and are familiar with the
+process. Basic programming knowledge and familiarity with the Linux command
+line are expected.
+
+## Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html) Required to build **Virtcontainers pause**
+
+## Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers).
+Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+There are two issues to enable ``--version`` option in ``Proxy`` and ``Shim``.
+This way the end user can know for sure which version of these components are
+installed and running.
+
+  * [https://github.com/clearcontainers/proxy/issues/69](https://github.com/clearcontainers/proxy/issues/69)
+  * [https://github.com/clearcontainers/shim/issues/32](https://github.com/clearcontainers/shim/issues/32)
```

Comment:
FYI (not required in this doc I don't think), I've also raised issues for the CC 2.1 versions of the shim and proxy:

- https://github.com/01org/cc-oci-runtime/issues/981
- https://github.com/01org/cc-oci-runtime/issues/982